### PR TITLE
Add page to add interface

### PR DESCRIPTION
**Pull request by PouuleT**
Adds an OTB interface manually and have all the right config ( macvlan +
ip4table )

### CODE REVIEW

File: overthebox/files/etc/udhcpc.user [hunk 1]
```diff
@@ -1,93 +1,174 @@
...
+		commit network
+		EOF
+	fi
+}
+
+_check_interface() {
+	original_ip="$ip"
+
+	# We need to check that the interface is given a "non used" ip address
+	while ip addr | grep 'inet ' | grep -sq "$ip"; do
```
Comment (gregdel) [inline overthebox/files/etc/udhcpc.user]: ```ip -4 -o addr | grep ...``` ?

File: overthebox/files/etc/udhcpc.user [hunk 2]
```diff
@@ -1,93 +1,174 @@
...
+}
+
+_get_other_ip() {
+	# Calculate new ip
+	ip=$(_next_ip "$ip")
+}
+
+_next_ip() {
+	# TODO: Handle this better
+	echo "$1" | awk -F. '{ print $1"."$2"."$3"."$4+1 }'
```
Comment (gregdel) [inline overthebox/files/etc/udhcpc.user]:
```
❯ printf "%d.%d.%d.%d" 1 2 3 $((4 + 1))
1.2.3.5
```

File: luci-app-overthebox/luasrc/controller/admin/overthebox.lua [hunk 3]
```diff
@@ -50,6 +51,64 @@ function index()
...
+    ucic:set("network", devID, "type", "macvlan")
+    ifname = ifID
+  end
+
+  -- Create the interface
+  ucic:set("network", ifID, "interface")
+  ucic:set("network", ifID, "ifname", ifname)
+  ucic:set("network", ifID, "proto", protocol)
+  ucic:set("network", ifID, "label", interface_name)
+  ucic:set("network", ifID, "metric", id + 10)
```
Comment (gregdel) [inline luci-app-overthebox/luasrc/controller/admin/overthebox.lua]: We shouldn't use the same base index
If you let the `dhcpc.user` script create the first interface and you create the second one by hand, you'll end up with the same metric and ip4table: 10.
This fucks everything :P

Comment (PouuleT) [inline luci-app-overthebox/luasrc/controller/admin/overthebox.lua]: metric was good, but ip4table was a duplicate you're right!
Fixed!

**Comment by gregdel** [comment]
Waiting for @angt last review. You can remove the WIP label ;)

**Pull request merged**